.github/workflows/static.yml

```diff
@@ -1,43 +1,39 @@
 name: golang/static
 on: pull_request
-
 jobs:
-
   vet:
     name: vet
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
-    - name: vet
-      uses: grandcolline/golang-github-actions@v1.1.0
+    - uses: actions/setup-go@v2
+    - uses: actions/checkout@v2
       with:
-        run: vet
-        comment: false
-  
-  # TODO: Enable linters below one by one and fix issues with each
-
+        path: repo
```

Comment:
The default working directory is the checked out repository, we can remove all the `repo` references

Reply:
Thanks, @HenryNguyen5 , that cleaned things up a bit.